### ISSUE TITLE
Suggestion: More Advanced Rocketry Interactions

### ISSUE THREAD

**Issue by AmishInvader** [FTBTeam/FTB-Interactions#512]
Add one or more gas giants to the planet list for gas mining from your space station.
Suggested gas mining yields:
-Gas Giant (Jupiter/Saturn)
hydrogen, helium, ammonia, helium 3, deuterium, methane, ethane, natural gas?

-Volcanic (like Io)
co2, sulfuric gas, sulfuric acid, sulfur dioxide, sulfuric naphtha, hydrochloric acid, hydrofluoric acid, ammonia

-Algae Ocean/Tropical (Primordial Earth)
co2, nitrogen, methane, chloromethane, hydrochloric acid

-Fun stuff that would be nice to have
dew of the void (ender io teleporter/rod of return fluid)
unstable mana
radon
fluorine
empowered oil
other rocket fuel recipes (dimethylhydrazine + dinitrogen tetroxide)

These could probably be gated by an artifact, or a complex unmanned vehicle assembler recipe, if you're worried about progression.

Maybe add some more interesting asteroid mining ores or orbital laser drill products from other planets/moon.

**Comment by Vyraal1** [comment]
I like the idea, but this one might be a long shot/low priority for the pack. Mostly because the satellite missions aren't just very fun, and really without having extremely gamebreaking rewards in my experience, it feels hard to make it justified or worth it. 

Thematically I think is a flavor win to use this - but the farthest we went was Orbital Miner because in the end, the asteroid mining machines so just boil down to GUI's rather than say controlling a ship itself, so the additional interactions aren't that interesting to me.

That being said, @Saereth would probably have a better vision for this.

**Comment by AmishInvader** [comment]
By satellite missions you mean the data obtaining system for finding asteroids? I agree it's not great.

Gas mining with the unmanned vehicles is a bit different though.  You just park a spacestation above a planet, supply rocketfuel, and send them off to collect a resource.  It requires some setup and logistics like anything else.

I don't really see how obtaining some of these things would be gamebreaking.  You could make rocketfuel from a vacuum freezer, setup a gas miner and get some helium 3.  Currently you can setup a vacuum freezer, and centrifuge helium 3 and deuterium straight from the air if you so desire.  

Petrochemicals can be made infinitely already with just a pyrolyse oven and some processing.  Most other gases/acids are just air, brine, and water, so it's not exactly gamebreaking, more of a step up from earlier processing which you had to do to reach space in the first place. (Which is kind of in the spirit of gregtech tiering)

That's my justification anyway, it wouldn't need to interfere with your current radon gating or whatever.  I've been enjoying the pack so far either way, nice job so far.

**Comment by Saereth** [comment]
satelite missions were added, may add more later but low piority
